.devcontainer/install-dev-tools.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Let's put this in the scripts directory rather than a hidden folder. Potentially we update the debug image as well to call it to standardise.

Reply:
I agree with you